lib/api/model/regions_meta.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'regions_meta.g.dart';
+
+@JsonSerializable()
+class RegionsMeta {
```

Comment:
Meta class can be generalized as its the same object for all paged lists

Reply:
Rename will be done in autocomplete branch